tests/example_app/example/about/story/templates/index.html

```diff
@@ -0,0 +1,17 @@
+{% extends "layout.html" %}
+
+{% block body %}
+  <a href="{{ url_for('about.story.index') }}">Our Story</a>
+  <a href="{{ url_for('about.team.index') }}">Our Team</a>
+
+  <h2>Our Story</h2>
+  This is an example application
```

Comment:
Can you change this to `<p>This is an example application.</p>` (with the period).